index.html

```diff
@@ -6,37 +6,31 @@
 		<title>Thrilling Frontend Technical Challenge</title>
 
 		<link rel="icon" type="image/x-icon" href="/tech-challenge/favicon.ico" />
+		<link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+		<script src="https://cdn.jsdelivr.net/npm/lodash@4.17.15/lodash.min.js"></script>
+		<script src="search.js"></script>
```

Comment:
consider adding [async](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async) or [defer](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer) attributes to prevent js from blocking page rendering, or moving `script` tags down to immediately before the closing `</body>` tag

Reply:
Great point.  It’s been awhile since I’ve manually added scripts to HTML index page and I forgot that rule of thumb.